pkg/api/v1alpha1/cluster_webhook.go

```diff
@@ -33,14 +33,11 @@ func (r *ScyllaCluster) SetupWebhookWithManager(mgr ctrl.Manager) error {
 		Complete()
 }
 
-// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
```

Comment:
Is this change part of running "make manifests"?

Reply:
no, i manually removed this comment. It was generated by the `kubebuilder` during project initializaiton.